packages/landingpage-new/components/layout/footer.tsx

```diff
@@ -5,17 +5,21 @@ import LinkItem from './linkItem';
 export default function Footer() {
   return (
     <footer className={styles.footer}>
-      {Routes.map(({ name, url, subRoutes }) => (
-        <div key={url} className={styles.col}>
-          <LinkItem url={url} name={name} />
-          {subRoutes.map(({ name: subRouteName, url: subRouteUrl }) => (
-            <LinkItem
-              key={subRouteName + subRouteUrl}
-              url={`${url}#${subRouteUrl}`}
-              name={subRouteName}
-              isDense={true}
-            />
-          ))}
+      {Routes.map(({ name: mainRouteName, url: mainRouteUrl, subRoutes }) => (
+        <div key={mainRouteUrl} className={styles.col}>
+          <LinkItem url={mainRouteUrl} name={mainRouteName} />
+          {subRoutes.map(({ name: subRouteName, url: subRouteUrl }) => {
+            const subrouteDivider =
```

Comment:
We might have other sub routes divided by `/` in the future so it would make sense to extract this logic in a function like this:

```tsx
const getDividerByMainRoute = (mainRoute: MainRoutes): '/' | '#' => {
  switch (mainRoute) {
    case MainRoutes.GETTING_STARTED:
      return '/';
    default:
      return '#';
  }
};
```